parsers/AR.py

```diff
@@ -471,6 +471,52 @@ def webparser(req):
 
     return data_table
 
+def fetch_price(country_code='AR', session = None):
+    """
+    Requests the last known power price of a given country
+    Arguments:
+    country_code (optional) -- used in case a parser is able to fetch multiple countries
+    session (optional)      -- request session passed in order to re-use an existing session
+    Return:
+    A dictionary in the form:
+    {
+      'countryCode': 'FR',
+      'currency': EUR,
+      'datetime': '2017-01-01T00:00:00Z',
+      'price': 0.0,
+      'source': 'mysource.com'
+    }
+      """
+    s = session or requests.Session()
+    price_req = s.get(cammesa_url)
+    psoup = BeautifulSoup(price_req.content,'html.parser')
+    find_price = psoup.find('td', class_ = "cssFuncionesLeft", align = "left")
+    
+    try:
+        price_text = find_price.getText()
+
+        #Strip all whitespace and isolate number.  Convert to float.
+        price_nws = "".join(price_text.split())
+        lprice = price_nws.rpartition(':')[2]
+        rprice = lprice.split('[')[0]
+        price = float(rprice.replace(',','.'))
+    
+    except AttributeError, ValueError:
+        #Price element not present or no price stated.
+        price = None
+    
+    datetime = arrow.now('UTC-3').datetime
```

Comment:
The source has a timestamp for demand but price has none.  I suspect that the price does not change very often (every few months) hence using 'now' time.  If you take a look at the [source](http://portalweb.cammesa.com/default.aspx) then maybe you can find a better time to use.  Happy to code it however you want.